docgen/main.go

```diff
@@ -0,0 +1,153 @@
+// Generate markdown from the JSON schema for the Consul ECS config.
+package main
```

Comment:
Can you put a comment at the top here (or a readme in this dir) that gives a brief description of what this script does and how to run it.